<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LedControlWidget</class>
 <widget class="QWidget" name="LedControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout2">
     <item>
      <widget class="QComboBox" name="comboBoxMode"/>
     </item>
     <item>
      <widget class="QLabel" name="labelFrequency">
       <property name="text">
        <string>Freq:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxFrequency"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ledColorButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxRed"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxGreen"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxBlue"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
